bool blue_color
bool green_color
bool white_color